java/client/build.gradle

```diff
@@ -134,26 +123,58 @@ tasks.register('testFfi', Test) {
 }
 
 tasks.register('buildAll') {
-    dependsOn 'protobuf', 'buildRust', 'testFfi'
+    dependsOn 'protobuf', 'buildRustRelease', 'testFfi'
     finalizedBy 'build'
 }
 
 compileJava.dependsOn('protobuf')
 clean.dependsOn('cleanProtobuf', 'cleanRust')
-test.dependsOn('buildRust')
```

Comment:
Does the test task no longer build rust or does it still happen even when we remove this line?

Reply:
We're using release builds, so the buildRust task is not needed (it builds debug only).

Reply:
but we still want to keep non-release build for development